psi4/driver/procrouting/proc.py

```diff
@@ -3008,7 +3008,7 @@ def run_cc_property(name, **kwargs):
     if n_one > 0:
         # call oe prop for GS density
         oe = core.OEProp(ccwfn)
-        oe.set_title("CC")
+        oe.set_title(name.upper())
```

Comment:
Are the generic `CC DIPOLE` still getting set? Used anywhere in the tests?

Reply:
Just caught this, it is used in ctest 46, and yes this is failing. I'll probably change `ccdensity.cc` to explicitly use the old `CC DIPOLE/QUADRUPOLE` for proc::env, as wrong as it feels. Generic `CC DIPOLE/QUADRUPOLE` are not being set on wfn.

Reply:
actually, this is done at in OEProp, e.g. for the dipole: https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/oeprop.cc#L1226
OEProp is nicely set up to use a consistent naming structure. do I intentionally break this with an `if` statement for CC to preserve the old names, or let it use the full method name and update the only test that uses it? (which leaves the unfortunate issue of _response_ property core variables having generic `CC` names, while _OEProp_ properties have the full method title... might be okay to let this go if proc::env is not long for this world)